packages/odyssey-react-mui/src/labs/datePickerTheme.tsx

```diff
@@ -145,13 +145,15 @@ export const datePickerTheme: ThemeOptions = {
       styleOverrides: {
         root: {
           display: "block",
+          // Matches Popper width
+          width: "calc(100vw - 64px)",
+          maxWidth: "55ch",
```

Comment:
Where's that popper width stored? Can we use a value from somewhere else?

Reply:
This value doesn't exist anywhere. This was just one of those weird things I needed to do to overcome weirdness from MUI

Reply:
I've seen this number in other places, unless I'm wrong. Can we comment that so it's clear why there's a magic number?